go-controller/pkg/ovn/master.go

```diff
@@ -253,6 +253,17 @@ func (oc *Controller) StartClusterMaster(masterNodeName string) error {
 		_ = oc.nodeLocalNatIPv6Allocator.Allocate(net.ParseIP(types.V6NodeLocalDistributedGWPortIP))
 	}
 
+	// Enable logical datapath groups on OVN 20.12 and later; option is
+	// ignored if OVN doesn't understand it. Logical datapath groups reduce
+	// the side of the southbound database in large clusters. ovn-controllers
+	// should be upgraded to a version that supports them before the option
+	// is turned on by the master.
+	_, stderr, err := util.RunOVNNbctl("set", "NB_Global", ".", "options:use_logical_dp_groups=true")
```

Comment:
Does it make sense to use `NBGlobalSetOptions` API of go-ovn bindings here?

Reply:
@abhat sure, why not :)